dependencies/windows/install-dependencies.cmd

```diff
@@ -57,11 +60,11 @@ if not exist gnudiff (
   del "%GNUDIFF_FILE%"
 )
 
-if not exist gnugrep (
+if not exist gnugrep\%GNUGREP_VERSION% (
   wget %WGET_ARGS% "%BASEURL%%GNUGREP_FILE%"
-  mkdir gnugrep
+  mkdir gnugrep\%GNUGREP_VERSION%
   echo Unzipping %GNUGREP_FILE%
-  unzip %UNZIP_ARGS% "%GNUGREP_FILE%" -d gnugrep
+  unzip %UNZIP_ARGS% "%GNUGREP_FILE%" -d gnugrep\%GNUGREP_VERSION%
```

Comment:
This isn't quite right, because this actually unzips the files to `gnugrep/3.0/gnugrep-3.0/grep.exe` instead of to `gnugrep/3.0/grep.exe` directly

Reply:
Fixed by reuploading the zip, removing the extra level of directory structure